core/src/main/java/org/stellar/anchor/util/UrlValidationUtil.java

```diff
@@ -8,10 +8,16 @@
 
 public class UrlValidationUtil {
   static UrlConnectionStatus validateUrl(String urlString) {
+    return validateUrl(urlString, false);
```

Comment:
Do we always want to set this to false? Can we make this specified in configuration or env var maybe?

Reply:
Is it kind of recursion if we make this configurable?

Reply:
I think we can make the choice to the decision of the validator of each config.